content/en/tracing/setup_overview/custom_instrumentation/ruby.md

```diff
@@ -157,19 +154,19 @@ custom_error_handler = proc do |span, error|
   span.set_error(error) unless error.message.include?("a special exception")
 end
 
-Datadog.tracer.trace('example.trace', on_error: custom_error_handler) do |span|
+Datadog::Tracing.trace('example.trace', on_error: custom_error_handler) do |span|
   example_method()
 end
 ```
 
 ## Adding spans
 
-If you aren't using supported library instrumentation (see [library compatibility][4]), you can manually instrument your code. Add tracing to your code by using the `Datadog.tracer.trace` method, which you can wrap around any Ruby code:
+If you aren't using supported library instrumentation (see [library compatibility][4]), you can manually instrument your code. Add tracing to your code by using the `Datadog::Tracing.trace` method, which you can wrap around any Ruby code:
 
-To trace any Ruby code, you can use the `Datadog.tracer.trace` method:
+To trace any Ruby code, you can use the `Datadog::Tracing.trace` method:
 
 ```ruby
-Datadog.tracer.trace(name, options) do |span|
+Datadog::Tracing.trace(name, options) do |span|
   # Wrap this block around the code you want to instrument
   # Additionally, you can modify the span here.
   # for example, change the resource name, or set tags
```

Comment:
⬇️  In line 196 `child_of` is no longer a supported option. It also doesn't mention `type` or `continue_from`.

(Should we perhaps link to a more up-to-date part of the docs rather than keeping this copy here?)